src/shapes/Textbox.ts

```diff
@@ -4,6 +4,7 @@ import { IText } from './IText/IText';
 import { classRegistry } from '../ClassRegistry';
 import { createTextboxDefaultControls } from '../controls/commonControls';
 import { JUSTIFY } from './Text/constants';
+import type { TextStyleDeclaration } from 'fabric/node';
```

Comment:
Second time makes me itch
I will PR that fix probably...

Reply:
fixed